source/fundamentals/monitoring.txt

```diff
@@ -148,17 +148,15 @@ The driver defines nine SDAM events. The driver divides these nine events
 between three separate listener interfaces which each listen for three of the
 nine events. Here are the three interfaces and the events they listen for:
 
-- ``ClusterListener``: :spec:`topology </server-discovery-and-monitoring/server-discovery-and-monitoring.rst#topology>`
-  related events
+- ``ClusterListener``: topology related events
```

Comment:
S: [hyphen for clarity (compound modifier)](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#hyphens-in-compound-modifiers)

```suggestion
- ``ClusterListener``: topology-related events
```